### DISCUSSION TITLE
Command to send to UG22 for the burner

### DISCUSSION THREAD

**Discussion by smaca69** [artisan-roaster-scope/artisan#1137] [General]
Hi,

I wanted to create custom buttons that can change the burner setting by +/- 5.  What would be the command I need to send to the UG22 to be able to do this?

**Comment by smaca69** [comment]
To explain a little more, I wanted to create 2 buttons "Gas +5" and "Gas -5". Every time I press either of the buttons, it will either reduce or increase the current burner value by +/- 5%

**Reply by smaca69** [reply]
All good, figured it out!!  Just want to know how do you make the button stay in its original state/colour and not pressed?

**Reply by MAKOMO** [reply]
Maybe you can document your solution here with a screenshot? The last button pressed is indicated by inverting its color. There is no way to have buttons keep their color.

**Reply by smaca69** [reply]
How to create buttons that allow you to either increase or reduce the current burner value by +/-5% on a UG22C

1. From the menu, select Config-->Events
2. Select the "Buttons" tab
3. Click on the "Add" button to add a new button
4. Add the button label (e.g. Gas -5%)
5. Set the button type to +-Burner
6. Set the value to either a +ve or -ve number depending on whether you want to increase or decrease the burner (e.g. -5)
7. Set the Action to "S7 Command"
8. Enter the following text in the Documentation section --> SetDBint(200,80,{})
9. Set Visibility to ON
10. Set your desired button color

The S7 command "SetDBint(200,80,{})" will work for any value as it substitutes the parentheses with the value you set in Step 6.

Refer to image below

![image](https://user-images.githubusercontent.com/20550292/235424512-9820bd1a-3e05-4a20-9b84-0e4de280cfd6.png)
